.github/workflows/test.yml

```diff
@@ -205,6 +206,86 @@ jobs:
           file: ./coverage.txt
         if: "env.GIT_DIFF != ''"
 
+  test-race-1:
+    runs-on: ubuntu-latest
+    needs: split-test-files
+    timeout-minutes: 15
+    steps:
+      - uses: actions/checkout@v2
+      - uses: technote-space/get-diff-action@v3
+        id: git_diff
+        with:
+          SUFFIX_FILTER: |
+            .go
+            .mod
+            .sum
+      - uses: actions/download-artifact@v2
+        with:
+          name: "${{ github.sha }}-aa"
+        if: "env.GIT_DIFF != ''"
+      - name: Run tests with race detector
+        run: cat xaa.txt | xargs go test -mod=readonly -timeout 15m -race -tags='cgo ledger test_ledger_mock'
```

Comment:
cant run tests if it has all been skipped. Tests will always fail if no go files have been touched here